opus/application/apps/ui/views.py

```diff
@@ -505,9 +505,17 @@ def api_init_detail_page(request, **kwargs):
     new_products = OrderedDict()
     for version in products:
         new_products[version] = OrderedDict()
+        selection = None
         for product_type in products[version]:
             file_list = products[version][product_type]
             product_info = {}
+            if (selection is None and (product_type[3].find('Browse Image') or
```

Comment:
If there's room on the line, put the AND clauses on different lines:

```
if (X and
    (Y or Z)):
```

If not, then:

```
if (X and
    (Y or
     Z)):
```